app/config/AppConfig.scala

```diff
@@ -62,6 +62,27 @@ class AppConfig @Inject()(
   lazy val classificationHelpUrl = loadConfig("urls.classificationHelp")
   lazy val ecicsToolUrl = loadConfig("urls.ecicsTool")
 
+  lazy val customsDeclarationsInformation = servicesConfig.baseUrl("customs-declarations-information")
+  lazy val fetchMrnStatus = servicesConfig.getConfString(
```

Comment:
Could maybe have a helper method to get services config or throw exception - like `loadConfig`?

Reply:
as discussed with Steve, this may not be necessary

Reply:
True - it was not a blocker

Reply:
I would check here is address is correct - ends with slash